hazelcast/src/main/java/com/hazelcast/internal/monitors/HealthMonitor.java

```diff
@@ -381,6 +380,8 @@ private void renderHeap() {
                     .append(numberToUnit(runtimeUsedMemory.read())).append(", ");
             sb.append("heap.memory.free=")
                     .append(numberToUnit(runtimeFreeMemory.read())).append(", ");
+            sb.append("heap.memory.available=")
```

Comment:
`runtimeAvailableMemory` was unused, so either we add it here or we should remove it.